wger/settings_global.py

```diff
@@ -370,3 +386,8 @@
     'EMAIL_FROM': 'wger Workout Manager <wger@example.com>',
     'TWITTER': False
 }
+
+SOCIAL_AUTH_FACEBOOK_KEY = os.getenv('FB_KEY')
+SOCIAL_AUTH_FACEBOOK_SECRET = os.getenv('FB_SECRET')
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.getenv('GOOGLE_KEY')
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = os.getenv('GOOGLE_SECRET')
```

Comment:
Add a single line at the end of the file.

Reply:
Done